src/nupic/bindings/algorithms.i

```diff
@@ -1778,3 +1814,10 @@ inline PyObject* generate2DGaussianSample(nupic::UInt32 nrows, nupic::UInt32 nco
     return py_s.close();
   }
 }
+
+%ignore nupic::algorithms::temporal_memory::TemporalMemory::getActiveCells;
```

Comment:
The %ignore operator in SWIG tells the parser to ignore any declarations that follow it. Note that I've moved the %include below the ignore, which has the effect of not creating bindings for the declarations of `getActiveCells`, etc. in `TemporalMemory.hpp`. We want this so that SWIG creates bindings for the version of `getActiveCells`, etc. that I defined above, which return lists, rather than the default bindings, which return tuples.

Finding the correct way to override a binding with the same name as that used in C++ took a fair bit of investigation, but I believe that this is the correct approach. In general, the approach is:

```C++
// First you define your bindings function
PyObject* myFunction()
{
  ...
}

// %ignore ignores matching declarations *after* it is called
%ignore myFunction;

// Finally the include needs to be at the end after both the binding definition and the ignore
%include "header.hpp"
```